Bindings/bindings.json

```diff
@@ -383,6 +383,9 @@
                         "capability": "blob"
                     }
                 }
+            ],
+            "filters": [
+                "showHiddenBinding"                
```

Comment:
showHiddenBinding is the query string key you need to add show ApiHub Binding. It is hidden by default when the filter setting is present. My understanding was that we wanted hide API Hub File bindings.